spec/requests/admin/backups_controller_spec.rb

```diff
@@ -248,11 +248,102 @@ def map_preloaded
           resumableCurrentChunkSize: '1',
           file: fixture_file_upload(Tempfile.new)
         }
+        expect_job_enqueued(job: :backup_chunks_merger, args: {
+          filename: filename, identifier: 'test', chunks: 1
+        }, at: 5.seconds.from_now)
 
         expect(response.status).to eq(200)
         expect(response.body).to eq("")
       end
     end
+
+    describe "completing an upload by enqueuing backup_chunks_merger" do
+      let(:filename) { 'test_Site-0123456789.tar.gz' }
```

Comment:
This PR looks good I just want to confirm this test is reasonably fast?

Reply:
Yes, even the one with 3 POST calls is `Finished in 0.49878 seconds`. It's just uploading Tempfiles which are 0 bytes.